metering/reports/metering-about-reports.adoc

```diff
@@ -5,9 +5,8 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-A Report is an API object that provides a method to manage periodic ETL (Extract Transform and Load) jobs using SQL queries.
-They are composed using other Metering resources such as `ReportQueries`, which provide the actual SQL query to run, and `ReportDataSources`, which are what define the data available to the ReportQueries and Reports.
+A `Report` custom resource provides a method to manage periodic Extract Transform and Load (ETL) jobs using SQL queries. Reports are composed from other metering resources such as report queries that provide the actual SQL query to run, and report data sources that define the data available to the report queries and reports.
```

Comment:
Does this need to be `such as ReportQuery resources that provide ...`? The same would apply to the other metering-related custom resources mentioned in this sentence.